components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -172,94 +229,45 @@ open class BrowserMenu internal constructor(
 }
 
 @VisibleForTesting
-internal fun PopupWindow.displayPopup(
```

Comment:
I refactored this method since this is when the menu position would be calculated (in this method and the ones called by it)
But I needed that position sooner since the menu should be expandable only when placed at the bottom.

This method is called only once. Keeping the old version and choose between them depending on if the menu should be expandable would mean duplicating a lot of code (also re-executing some calculations) with really no difference in the end functionality.